documents/models.py

```diff
@@ -95,8 +96,7 @@ def votes(self):
     @property
     def is_certified(self) -> bool:
         """Is the document tagged with the "officiel" tag ?"""
```

Comment:
The docstring is not valid anymore

Reply:
Also, the indirection is not really useful, let's refactor and remove `is_certified` as we have `. certified` now :)